README.md

```diff
@@ -535,19 +561,19 @@ The Cypress GH Action does not spawn or create any additional containers - it on
 
 ### Component tests
```

Comment:
@robertguss We need to add 1 or 2 examples for v10 that have component testing running in this repo so that the `component: true` attribute is used and tested.

Reply:
@kevinold I have added the CRA with Chakra UI project and created an example-component-test.yml workflow for it.

Reply:
@kevinold I commented out the component test workflow for now since it depends upon code and updates in this PR. Once we merge this PR, we will need to uncomment it to get it working.

Reply:
@robertguss Not sure what you mean, as we want those to be working as this PR is merged.  Let me review and see if I can understand, but we might need to sync on it.

Reply:
@robertguss Testing this to see if it allows the CT example to run https://github.com/cypress-io/github-action/pull/503/commits/3e9a6baf78587adf47bc55ac99382c36b2233989

Reply:
@robertguss It worked! https://github.com/cypress-io/github-action/runs/5208533374?check_suite_focus=true

Reply:
@kevinold awesome! I could have sworn I tried this before, but glad to know it works ;)

Reply:
yeah I tried it in this commit, but it didn't seem to work: https://github.com/cypress-io/github-action/pull/503/commits/4feb2488c38020bb1c472043bdfc3892719b0f3e